### PR TITLE
Add message history related mappings.

### PR DESCRIPTION
**Pull request by occanowey**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/hud/ChatHud.mapping [hunk 1]
```diff
@@ -10,12 +11,15 @@ CLASS cwu net/minecraft/client/gui/hud/ChatHud
 		ARG 1 x
 		ARG 3 y
 	METHOD a draw (I)V
+	METHOD a addMessageHistory (Ljava/lang/String;)V
```
Comment (asiekierka) [inline mappings/net/minecraft/client/gui/hud/ChatHud.mapping]: addMessageHistory sounds weird

Comment (occanowey) [reply mappings/net/minecraft/client/gui/hud/ChatHud.mapping]: Maybe addToMessageHistory?

**Comment by liach** [comment]
does this field exist in command block gui? consider mapping if it does.

**Comment by occanowey** [comment]
It doesn't look like command blocks have command history.

**Review by liach — APPROVED** [review]

**Pull request merged**